lib/reportAProblem/submitEmail.js

```diff
@@ -0,0 +1,50 @@
+import "node-fetch";
+
+export const submitEmail = async (
+  data,
+  templateId,
+  email,
+  notifyApiAddress,
+  notifyApiKey
+) => {
+  const notifyResponse = await fetch(notifyApiAddress, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `ApiKey-v1 ${notifyApiKey}`,
+    },
+    body: JSON.stringify({
+      email_address: email,
+      template_id: templateId,
+      // notify requires all values in the template to be present
+      // in the request.... highly annoying but providing defaults solves this
+      personalisation: {
+        page_name: "unknown",
+        language: "unknown",
+        incorrect_information: "no",
+        incorrect_information_details:
+          "No incorrect information was reported on the page.",
+        unclear_information: "no",
+        unclear_information_details:
+          "No unclear information was reported on the page.",
+        info_not_found: "no",
+        info_not_found_details:
+          "The user did not report not being able to find what they were looking for.",
+        adaptive_technology: "no",
+        adaptive_technology_details:
+          "The user did not report any issues with adaptive technologies accommodation.",
+        privacy_issues: "no",
+        privacy_issues_details: "The user did not report any privacy issues",
+        no_where_else_to_go: "no",
+        no_where_else_to_go_details:
+          "The user did not report not knowing where to provide feedback",
+        other: "no",
+        other_details:
+          "There is no unspecified issues that the user would like to report",
+        ...data,
```

Comment:
I rewrote the `*_details` fields locally to pass in empty strings if nothing was submitted. We could just say "N/A" or something here since it just means that no details were passed in.

Reply:
hmmmm but then you would have a series of N/A's with no headings ?